### ISSUE TITLE
[BUG] Update from 1.7.14 > 1.7.15, huge amounts of the Spectrum blocks and items have disappeared

### ISSUE THREAD

**Issue by GeekyBakerBen** [DaFuqs/Spectrum#444]
**Describe the bug**
I just tried updating my 1.20.1 Minecraft server from 1.7.14 to 1.7.15, and a HUGE amount of the previously present Spectrum content has just...disappeared. Pigments, coloured woods, saplings, most of the blocks used to craft the shrine structures etc. This has caused most of my world to break and disappear and I'm going to have to roll back to a previous backup and revert to 1.7.14.

If this is intentional changes (through some big mod overhaul), you really ought to write that into all the descriptions and patch notes, so people are aware their worlds will break.

**To Reproduce**
Update from 1.7.14 to 1.7.15, on 1.20.1 Fabric Minecraft

**Expected behavior**
I'm assuming it's a result of block renames or something similar but, I'd kind of expect most of my world to be intact, and if this really is intended, at least put plenty of warnings on all your content pages, like other mods do when they have a major overhaul.

**Minecraft version**
1.20.1 Fabric

**Mod version**
1.7.14 -> 1.7.15

**Comment by imreallybadatnames** [comment]
cc: @DaFuqs

**Comment by DaFuqs** [comment]
That was only a small update and did not touch any block/item ids at all, actually.
Do you have the log from when you did the update? I upgraded my worlds just fine.

**Comment by GeekyBakerBen** [comment]
I did suspect it seemed to be a big break for a seemingly simple update - I've rolled back, taken a look at the server and seen what happened - For some reason, when I uploaded the 1.7.15 update to the server, it didn't take and my server disconnected before the files uploaded fully, so what I was seeing was the results when you have 1.7.15 on Client side, and absolutely nothing Spectrum on the server xS

My bad! I'm sorry for any panic caused. I've now rolled back and then re-applied the .15 update and it seems to be fine so far.

I don't know if I should give you kudos for a mod that technically allows the server to work when only Client-side, and not crash out saying the files aren't present! xD
